src/components/Popups/ScheduleSetupPopup/ScheduleSetupPopup.tsx

```diff
@@ -0,0 +1,84 @@
+import { useEffect } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useNavigate } from 'react-router-dom';
+
+import { Modal } from 'components/Popups';
+import { Table, UiType } from 'components/Tables';
+import { StyledModalWrapper } from 'styles/styledComponents/Modal';
+import { StyledBodyLarge } from 'styles/styledComponents/Typography';
+import theme from 'styles/theme';
+import { APPLET_PAGES } from 'utils/constants';
+
+import { ScheduleSetupPopupProps } from './ScheduleSetupPopup.types';
+import { headCells } from './ScheduleSetupPopup.const';
+
+export const ScheduleSetupPopup = ({
+  popupVisible,
+  setPopupVisible,
+  tableRows,
+  chosenAppletData,
+  setChosenAppletData,
+}: ScheduleSetupPopupProps) => {
+  const { t } = useTranslation('app');
+  const navigate = useNavigate();
+
+  const handlePopupClose = () => {
+    setChosenAppletData(null);
+    setPopupVisible(false);
+  };
+  const handleBackClick = () => setChosenAppletData(null);
+
+  const handlePopupSubmit = () => {
+    setPopupVisible(false);
+    navigate(`/${chosenAppletData?.appletId}/${APPLET_PAGES.schedule}`);
+  };
+
+  const showSecondScreen = chosenAppletData && !chosenAppletData.hasIndividualSchedule;
+
+  useEffect(() => {
+    if (chosenAppletData?.hasIndividualSchedule) {
+      handlePopupSubmit();
+    }
+  }, [chosenAppletData]);
+
+  return (
+    <Modal
+      open={popupVisible}
+      onClose={handlePopupClose}
+      onSubmit={handleBackClick}
+      title={t('individualScheduleSetup')}
+      buttonText={showSecondScreen ? t('back') : ''}
+      hasSecondBtn={Boolean(showSecondScreen)}
+      secondBtnText={t('yes')}
+      onSecondBtnSubmit={handlePopupSubmit}
+      actionsAlign="end"
+      width="66"
+    >
+      <StyledModalWrapper>
+        {showSecondScreen ? (
+          <StyledBodyLarge
+            sx={{ marginTop: theme.spacing(-1) }}
+            dangerouslySetInnerHTML={{
```

Comment:
Anna suggested using the Trans component - I suppose it would be more correct.. please, take a look - https://github.com/ChildMindInstitute/mindlogger-admin/pull/96/files#diff-7b448268cd59e686fb8af34e9c61d38b2c06fb8eee29ae1a24c5b25dd44ad486R115

Reply:
Done